ocpp/messages.py

```diff
@@ -50,50 +61,71 @@ def pack(msg):
     return msg.to_json()
 
 
-def get_schema(name):
+def get_schema(message_type_id, action, ocpp_version):
     """
     Read schema from disk and return in. Reads will be cached for performance
     reasons.
+
     """
-    if name in _schemas:
-        return _schemas[name]
+    if ocpp_version == "1.6":
```

Comment:
we could do some string operations here and substitute the if elses clauses by:

```python
schemas_dir = "v" + ocpp_version.replace(".", "")
```
so then if new versions come we dont need to do a train of if clauses
what do you think?

Reply:
for the error raise, maybe we can just check if the ocpp_version is in the list ["1.6", "2.0", ...]

Reply:
Sure, I like it.

Reply:
Sure, I like it.